recipes/src/styles.js

```diff
@@ -0,0 +1,281 @@
+import styled, { createGlobalStyle } from 'styled-components';
```

Comment:
You didn't have to make this mobile responsive but it's nice you used media queries for the exercise.

Reply:
Yeah - I saw a blog post on it and wanted to try it.
Also, I may have been influenced by my initial decision to make it a single-page app. 😉